test/Microsoft.ML.Tests/TrainerEstimators/SymSgdClassificationTests.cs

```diff
@@ -33,7 +33,7 @@ public void TestEstimatorSymSgdInitPredictor()
             (var pipe, var dataView) = GetBinaryClassificationPipeline();
             var transformedData = pipe.Fit(dataView).Transform(dataView);
 
-            var initPredictor = ML.BinaryClassification.Trainers.StochasticDualCoordinateAscent().Fit(transformedData);
+            var initPredictor = ML.BinaryClassification.Trainers.StochasticDualCoordinateAscentCalibrated().Fit(transformedData);
```

Comment:
>StochasticDualCoordinateAscentCalibrated [](start = 65, length = 40)

why it got changed?

Reply:
Old SDCA without specifying loss function is the new SDCALR.

---
In reply to: [256211987](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256211987) [](ancestors = 256211987)

Reply:
Since we name SDCA+LR as `StochasticDualCoordinateAscent` now, this change will be reverted.

---
In reply to: [256219608](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256219608) [](ancestors = 256219608,256211987)